demos/can-component/accordion.html

```diff
@@ -106,9 +107,9 @@
 
 out.addEventListener("click", function(ev){
 	var el = ev.target;
-	var parent = el.parentNode;
+	var parent = canViewModel(el.parentNode);
 	if (el.nodeName === "BUTTON") {
-		parent.setAttribute("title", "Users");
+		parent.title = "Users";
```

Comment:
This demo is used in [can-component’s ViewModel docs](https://canjs.github.io/next/doc/can-component.prototype.ViewModel.html#Valuespassedfromattributes) to show how you can change a component’s element attributes to update the view model, but I don’t think this is supported in 4.0, so the demo now shows using `can-view-model` to get the component’s view model and set a property.